test/system/255-auto-update.bats

```diff
@@ -234,6 +234,8 @@ function _confirm_update() {
     _confirm_update $cname $ori_image
 }
 
+# This test can fail in dev. environment because of SELinux.
+# quick fix: chcon -t container_runtime_exec_t ./bin/podman
```

Comment:
This is already done by `hack/bats`, which is what you should be using for running bats tests...

Reply:
Thanks! I had no idea this script existed ... my muscle memory is stuck using `PODMAN=... bats ... [--filter ...]`